### PR TITLE
#66 Launch.java:55-57: Implement PUT operation to upload...

### PR DESCRIPTION
**Pull request by pinaf**
#66 Launch.java:55-57: Implement PUT operation to upload...

### CODE REVIEW

**Comment by karato** [comment]
Let me find a reviewer for this pull request, thanks for submitting it

**Comment by karato** [comment]
@darkled review it

File: src/main/java/com/nerodesk/Launch.java [hunk 1]
```diff
@@ -68,6 +68,9 @@
  * @todo #14:15min Add new section to the README described REST API for
  *  the file operations available. GET, PUT and DELETE operations should be
  *  described.
+ * @todo #66:30min Wire PUT method to file storage using TkPutFile. Add a test
+ * for saving content using the PUT method and a path matching TkPutFile.PATH.
+ * For example: PUT http://localhost:8080/api/files/filename.txt .
```
Comment (asaen) [inline src/main/java/com/nerodesk/Launch.java]: @pinaf please add leading spaces on lines 72-73 in order to create puzzle correctly

File: src/main/java/com/nerodesk/TkPutFile.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+            );
+        } catch (final IOException ex) {
+            Logger.info(
+                this,
+                String.format(
+                    "Internal error saving to storage at %s: %s",
+                    path,
+                    ex.getMessage()
+                )
+            );
```
Comment (asaen) [inline src/main/java/com/nerodesk/TkPutFile.java:89]: @pinaf let's use `com.jcabi.log.Logger.error(Object, String, Object...)` instead?

**Comment by asaen** [comment]
@pinaf looks nice, just few comments above.

**Comment by pinaf** [comment]
@darkled thank you. fixed.

**Comment by asaen** [comment]
@pinaf thank you
@rultor merge

**Comment by asaen** [comment]
@rultor merge

**Comment by pinaf** [comment]
@yegor256  @alevohin so what happens now?

**Comment by yegor256** [comment]
@pinaf it's outdated, as you see. there is a much simpler design in place already..

**Comment by pinaf** [comment]
@yegor256 yeah I waited for your merge. So what happens now?

**Comment by yegor256** [comment]
@pinaf the task will be closed and this pull request can be deleted

**Comment by pinaf** [comment]
@yegor256 got it.

**Comment by karato** [comment]
@darkled I added **18 mins** to your account, many thanks for working with the project!... you're getting extra minutes here (c=3)... +18 added to your rating, current score is: [+2068](http://www.netbout.com/b/35091?open=rating)

**Pull request closed without merging**